src/js/edu-benefits/1990e/config/form.js

```diff
@@ -117,11 +117,14 @@ const formConfig = {
               }
             }),
             'view:veteranId': _.merge(veteranId.uiSchema, {
-              vaFileNumber: {
-                'ui:title': 'Sponsor file number',
-              },
               veteranSocialSecurityNumber: {
                 'ui:title': 'Sponsor Social Security number'
+              },
+              'view:noSSN': {
+                'ui:title': 'Sponsor doesn\'t have a Social Security number',
```

Comment:
Should this be "I don’t know my sponsor’s Social Security number" (from your comment [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1744#issuecomment-288191320))?